kolibri/plugins/learn/assets/src/state/actions.js

```diff
@@ -706,7 +706,7 @@ function setAndSaveCurrentExamAttemptLog(store, contentId, itemId, currentAttemp
 }
 
 function closeExam(store) {
-  const examLog = store.state.examLog;
+  const examLog = Object.assign({}, store.state.examLog);
```

Comment:
Also, was manipulating the pageState directly causing any of the issues? Or are we doing this just cause